tests/testprogs/mountns_wrapper.c

```diff
@@ -0,0 +1,48 @@
+#define _GNU_SOURCE
+#include <sched.h>
+#include <sys/mount.h>
+#include <sys/stat.h>
+#include <libgen.h>
+#include <unistd.h>
+#include <string.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#define BUFFER_SIZE 255
+
+#define errExit(msg)    do { perror(msg); exit(EXIT_FAILURE); \
+                        } while (0)
+
+
+
+int main(int argc, char *argv[])
+{
+
+    char* private_mount = "/tmp/unshare/";
```

Comment:
I don't think this is unique enough to be run on peoples machines. `/tmp/bpftrace-unshare-test` is more likely to prevent conflicts and people will know where the file came from.

Reply:
I'll make sure it is unique too.

I don't think any of the other tests use /tmp as far as I know, if they do maybe a shared subfolder per test run could be a future standard?